tests/controllers/hibp.test.js

```diff
@@ -72,6 +72,40 @@ test("notify POST with breach, subscriber hash prefix and suffixes should call s
 });
 
 
+test("good notify POST with breach, secondary email hash prefix and suffixes should call sendEmail and respond with 200", async () => {
+  jest.mock("../../email-utils");
+  EmailUtils.sendEmail = jest.fn();
+  LocaleUtils.fluentFormat = jest.fn();
+
+  const testSecondaryEmail = "firefoxaccount-secondary@test.com";
+  const testSecondaryHash = sha1(testSecondaryEmail);
+  const testPrefix = testSecondaryHash.slice(0, 6).toUpperCase();
+  const testSuffix = testSecondaryHash.slice(6).toUpperCase();
+
+  HIBPLib.getBreachesForEmail = jest.fn();
+
+  const mockRequest = { token: AppConstants.HIBP_NOTIFY_TOKEN, body: { hashPrefix: testPrefix, hashSuffixes: [testSuffix], breachName: "Test" }, app: { locals: { breaches: testBreaches, AVAILABLE_LANGUAGES: ["en"] } } };
+  const mockResponse = { status: jest.fn(), json: jest.fn() };
+
+  await hibp.notify(mockRequest, mockResponse);
+
+  const mockFluentFormatCalls = LocaleUtils.fluentFormat.mock.calls;
+  expect (mockFluentFormatCalls.length).toBe(1);
+  const mockFluentFormatCallArgs = mockFluentFormatCalls[0];
+  expect (mockFluentFormatCallArgs[0]).toEqual(["en"]);
+  expect (mockFluentFormatCallArgs[1]).toBe("hibp-notify-email-subject");
```

Comment:
minor nit, but not sure if it'd be easier or more readable if we could destructure the `mockFLuentFormatCallArgs[]` to make these more readable.

```js
const [mockFluentLocales, mockFluentSubject] = mockFluentFormatCalls[0];
expect(mockFluentLocales).toEqual(["en"]);
expect(mockFluentSubject).toBe("hibp-notify-email-subject");
```